openmetadata-ui/src/main/resources/ui/src/components/TableProfiler/TableProfilerV1.tsx

```diff
@@ -404,23 +404,17 @@ const TableProfilerV1: FC<TableProfilerProps> = ({
                   />
                 )}
 
-                <Tooltip
-                  title={
+                <Link
+                  to={
                     editTest
-                      ? t('label.add-entity', {
-                          entity: t('label.test'),
-                        })
-                      : t('message.no-permission-for-action')
+                      ? getAddDataQualityTableTestPath(
+                          ProfilerDashboardType.TABLE,
+                          `${table?.fullyQualifiedName}`
+                        )
+                      : '#'
                   }>
-                  <Link
-                    to={
-                      editTest
-                        ? getAddDataQualityTableTestPath(
-                            ProfilerDashboardType.TABLE,
-                            `${table?.fullyQualifiedName}`
-                          )
-                        : '#'
-                    }>
+                  <Tooltip
+                    title={!editTest && t('message.no-permission-for-action')}>
```

Comment:
if the condition is not satisfying, will it be going to show a tooltip?

Reply:
@ShaileshParmar11  no it will not going to show us a tooltip